golang/cosmos/app/upgrade.go

```diff
@@ -231,6 +231,9 @@ func makeUnreleasedUpgradeHandler(app *GaiaApp, targetUpgrade string, baseAppLeg
 				vm.CoreProposalStepForModules(
 					"@agoric/builders/scripts/vats/upgrade-zcf.js",
 				),
+				vm.CoreProposalStepForModules(
+					"@agoric/builders/scripts/vats/upgrade-provisionPool.js",
```

Comment:
I don't see an a3p-integration test. Oh...

> Wallet provisioning is already covered in a3p-integration z:acceptance

But how can we tell whether this feature worked?